Filters/src/FilterCosmicsStage1_module.cc

```diff
@@ -36,6 +41,11 @@ namespace mu2e {
       using Name=fhicl::Name;
       using Comment=fhicl::Comment;
 
+      fhicl::OptionalAtom<art::InputTag> DSEnt {
```

Comment:
I am curious if the intent is to leave this always optional?  Or will it become required and being optional is transitional?

Reply:
Thanks of the clarification.